tests/util.h

```diff
@@ -223,6 +223,7 @@ class ProxyNode : public raftcpp::rpc::NodeService,
      * We may see the leader as source node forever?
      */
     void HandleRequestPushLogs(RpcConn conn, int64_t committed_log_index,
+                               int32_t pre_log_term_num,
```

Comment:
Should pre_log_term_num be pre_log_term_index?

Reply:
okey